index.html

```diff
@@ -1199,6 +1228,35 @@ <h3>Multistatus Correlation</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Status Message Alterations</h3>
+
+      <p>
+When a status list uses the status messages feature, it becomes possible for
+the <a>issuer</a> to increase the type of messages that are associated with
+the <a>verifiable credentials</a> it issues over time.
+      </p>
+
+      <p>
+This feature creates a potential privacy violation where <a>holder</a> of the
```

Comment:
Q. Is this status information about the subject or the issuee?
It cannot be about the holder since the holder can be anyone and is not a fixed entity.